### ISSUE TITLE
Some differences with swagger.io validator

### ISSUE THREAD

**Issue by murat11** [BigstickCarpet/swagger-parser#52]
Found some behavior which differs from swagger.io validator. 
Here is my [swagger.json](https://github.com/goalapi/specification/blob/25fb045251cb5c68acc5c09fa0fe6db4f17d0125/v1/schema/swagger.json).
[Your tool](http://bigstickcarpet.com/swagger-parser/www/index.html) validates it successfully, while  [swagger.io does not](http://online.swagger.io/validator/debug?url=https://raw.githubusercontent.com/goalapi/specification/25fb045251cb5c68acc5c09fa0fe6db4f17d0125/v1/schema/swagger.json)

May be the problem with [**header** in response](https://github.com/goalapi/specification/commit/25fb045251cb5c68acc5c09fa0fe6db4f17d0125#diff-aa3066403cb2ce81e969319b94c445dd). Because previous commit [validated successfully](http://online.swagger.io/validator/debug?url=https://raw.githubusercontent.com/goalapi/specification/09d96ecaff03824bb58ab0c741c9bf3128576391/v1/schema/swagger.json) by swagger.io

**Comment by BigstickCarpet** [comment]
yeah... I'm not sure why the swagger.io validator is returning an error.  Could be a bug in their validator, or I could be overlooking something wrong with your schema.
